catalog/spatial/registry/registry-ebrim-transformer/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -17,6 +17,26 @@
     <reference id="xmlParser" interface="org.codice.ddf.parser.Parser" filter="(id=xml)"
                availability="mandatory"/>
 
+    <bean id="registryObjectMetacardType" class="ddf.catalog.data.impl.MetacardTypeImpl">
+        <argument value="registry"/>
+        <argument>
+            <list>
+                <bean class="org.codice.ddf.registry.common.metacard.RegistryObjectMetacardType"/>
+                <bean class="ddf.catalog.data.impl.types.TopicAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.MediaAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.ContactAttributes"/>
+                <bean class="ddf.catalog.data.impl.types.CoreAttributes"/>
```

Comment:
Yeah, see, you're including the `Core` attributes here, so you can just use them.